src/algebra/big_operators/finprod.lean

```diff
@@ -408,14 +406,28 @@ begin
   simp [mul_support_mul]
 end
 
+@[to_additive] lemma finprod_inv_distrib {G : Type*} [comm_group G] (f : α → G) :
+  ∏ᶠ x, (f x)⁻¹ = (∏ᶠ x, f x)⁻¹ :=
+((mul_equiv.inv G).map_finprod f).symm
+
+lemma finprod_inv_distrib₀ {G : Type*} [comm_group_with_zero G] (f : α → G) :
+  ∏ᶠ x, (f x)⁻¹ = (∏ᶠ x, f x)⁻¹ :=
+((mul_equiv.inv₀ G).map_finprod f).symm
```

Comment:
Do you have `α : Type*` or `α : Sort*` here? If `Type*`, then please move these two lemmas to the previous location of `finprod_inv_distrib`.

Reply:
Then maybe the mul lemmas are in the wrong place too?

Reply:
Lemmas that assume something like `finite (mul_support f)` have to use `α : Type*` (because `mul_support f` is a `set`, and `set`s don't work with `Sort*`).

Reply:
I had ` α : Type*`; I have moved the lemmas to the previous location.